src/sets.jl

```diff
@@ -196,36 +195,54 @@ The advantage of the first format is the mapping between the `(i, j)` matrix ind
 Indeed,
 - the entry of matrix indices `(i, j)` has vectorized index `k = div((j-1)*j, 2) + i` if ``i \\leq j`` and `k = div((i-1)*i, 2) + j` if ``j \\leq i``;
 - and the entry with vectorized index `k` has matrix indices `i = isqrt(2k)` and `j = k - div((i-1)*i, 2)` or `j = isqrt(2k)` and `i = k - div((j-1)*j, 2)`.
-"""
-struct PositiveSemidefiniteConeTriangle <: AbstractVectorSet
-    dimension::Int
-end
-
-"""
-    PositiveSemidefiniteConeScaled(dimension)
 
-The (vectorized) cone of symmetric positive semidefinite matrices, with off-diagonals scaled.
-The entries of the upper triangular part of the matrix are given column by column (or equivalently, the entries of the lower triangular part are given row by row).
-An ``n \\times n`` matrix has ``n(n+1)/2`` lower-triangular elements, so for the vectorized cone of dimension ``d``, the corresponding symmetric matrix has side dimension ``\\sqrt{1/4 + 2 d} - 1/2`` elements.
-The off-diagonal entries of the matrices of both the cone and its dual are scaled by ``\\sqrt{2}`` and the scalar product is simply the sum of the pairwise product of the entries.
-
-### Examples
-
-The matrix
+### Duality note
+The scalar product for the symmetric matrix in its vectorized form is the sum of the pairwise product of the diagonal entries plus twice the sum of the pairwise product of the upper diagonal entries.
```

Comment:
I would add a reference to slide 6 of http://docs.mosek.com/slides/ismp2012/sdo.pdf, or even better to a relevant paper.